src/screens/SUE/SueListScreen.tsx

```diff
@@ -123,6 +181,13 @@ export const SueListScreen = ({ navigation, route }: Props) => {
                   label: 'Status',
                   name: 'status',
                   data: statuses
+                },
+                {
+                  type: FILTER_TYPES.DROPDOWN,
+                  label: 'Sortieren nach',
+                  name: 'sortBy',
+                  data: SORT_OPTIONS,
```

Comment:
the selection does not work properly. as you can see in your screenshot or in mine here:

![image](https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/c6043c0e-2413-4540-aeed-d062050af58a)

i have selected "betreff" but "Änderungsdatum" is marked